public/app/features/dashboard/api/utils.ts

```diff
@@ -1,12 +1,14 @@
-import { config } from '@grafana/runtime';
+import { config, locationService } from '@grafana/runtime';
 import { DashboardV2Spec } from '@grafana/schema/dist/esm/schema/dashboard/v2alpha0/dashboard.gen';
 import { DashboardDataDTO, DashboardDTO } from 'app/types';
 
 import { DashboardWithAccessInfo } from './types';
 
 export function getDashboardsApiVersion() {
+  const forcingOldDashboardArch = locationService.getSearch().get('scenes') === 'false';
```

Comment:
It is worth adding a test case in `utils.test.ts` to avoid breaking the fallback mechanism in the future since this is frequently used to debug scenes issues.

Reply:
added aff7a59bdb1c3523e430161e686282c4fafcf893 :)